lib/departure/option.rb

```diff
@@ -8,10 +8,7 @@ class Option
     # @param string [String]
     # @return [Option]
     def self.from_string(string)
-      pair = string.split('=')
-      name = pair[0][2..-1]
-      value = pair[1]
-
+      name, value = string.split(/\s|=/, 2)
```

Comment:
With this we're changing the behaviour of this class. `pair[0][2..-1]` was stripping the `--` out and now it's gone. The role of `.from_string` is to turn the command line input format into a key and its value without `--` or `-`. 

We basically strip the formatting out so that we can either read from CLI or pass a key and a value by hand to the constructor like in https://github.com/redbooth/departure/blob/master/lib/percona_migrator/cli_generator.rb#L17. This means we would only need to call `#normalize_option` when executing `#to_s`.